2023-10-15 02:42:25.539436: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2023-10-15 02:42:33.625840: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2023-10-15 02:42:34.241258: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: NVIDIA GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.582
pciBusID: 0000:03:00.0
2023-10-15 02:42:34.241329: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2023-10-15 02:42:34.642820: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10
2023-10-15 02:42:35.007093: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcufft.so.10
2023-10-15 02:42:35.611627: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcurand.so.10
2023-10-15 02:42:35.942995: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusolver.so.10
2023-10-15 02:42:36.385565: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusparse.so.10
2023-10-15 02:42:37.086588: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2023-10-15 02:42:37.087453: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2023-10-15 02:42:49.322596: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: NVIDIA GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.582
pciBusID: 0000:03:00.0
2023-10-15 02:42:49.322865: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2023-10-15 02:42:49.322906: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcublas.so.10
2023-10-15 02:42:49.322936: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcufft.so.10
2023-10-15 02:42:49.322965: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcurand.so.10
2023-10-15 02:42:49.322994: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusolver.so.10
2023-10-15 02:42:49.323024: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcusparse.so.10
2023-10-15 02:42:49.323053: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2023-10-15 02:42:49.323536: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2023-10-15 02:42:49.323586: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2023-10-15 02:42:50.986185: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2023-10-15 02:42:50.986225: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0 
2023-10-15 02:42:50.986233: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N 
2023-10-15 02:42:50.986857: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10406 MB memory) -> physical GPU (device: 0, name: NVIDIA GeForce GTX 1080 Ti, pci bus id: 0000:03:00.0, compute capability: 6.1)
2023-10-15 02:42:50.988199: I tensorflow/core/common_runtime/process_util.cc:115] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
Found 79063 non-validated image filenames.
Found 11148 non-validated image filenames.
Downloading data from https://github.com/keras-team/keras-applications/releases/download/densenet/densenet121_weights_tf_dim_ordering_tf_kernels_notop.h5
    8192/29084464 [..............................] - ETA: 0s 3153920/29084464 [==>...........................] - ETA: 0s 8445952/29084464 [=======>......................] - ETA: 0s12591104/29084464 [===========>..................] - ETA: 0s12705792/29084464 [============>.................] - ETA: 0s16900096/29084464 [================>.............] - ETA: 0s20922368/29084464 [====================>.........] - ETA: 0s20930560/29084464 [====================>.........] - ETA: 0s27975680/29084464 [===========================>..] - ETA: 0s29089792/29084464 [==============================] - 1s 0us/step
Traceback (most recent call last):
  File "/home/caap/LabelReliability_and_PathologyDetection_in_ChestXrays/Task-Pathology_detection/PD_save_models.py", line 162, in <module>
    model_acc1 = PD_save_models(json_name=path+"PD_model1.json", h5_name=path+"PD_model1.h5", lr=lr)
  File "/home/caap/LabelReliability_and_PathologyDetection_in_ChestXrays/Task-Pathology_detection/PD_save_models.py", line 106, in PD_save_models
    x = GlobalAveragePooling2D(keepdims=True)(input_tensor)
  File "/opt/itu/easybuild/software/TensorFlow/2.0.0-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/tensorflow_core/python/keras/layers/pooling.py", line 689, in __init__
    super(GlobalPooling2D, self).__init__(**kwargs)
  File "/opt/itu/easybuild/software/TensorFlow/2.0.0-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/tensorflow_core/python/training/tracking/base.py", line 457, in _method_wrapper
    result = method(self, *args, **kwargs)
  File "/opt/itu/easybuild/software/TensorFlow/2.0.0-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/tensorflow_core/python/keras/engine/base_layer.py", line 293, in __init__
    generic_utils.validate_kwargs(kwargs, allowed_kwargs)
  File "/opt/itu/easybuild/software/TensorFlow/2.0.0-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/tensorflow_core/python/keras/utils/generic_utils.py", line 599, in validate_kwargs
    raise TypeError(error_message, kwarg)
TypeError: ('Keyword argument not understood:', 'keepdims')
